front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -45,6 +45,12 @@ export const useStoreDataForSpeedLimitByTagSelector = ({ isStdcm } = { isStdcm:
     }
   }, [error]);
 
+  useEffect(() => {
+    if (isStdcm && !speedLimitByTag) {
+      dispatchUpdateSpeedLimitByTag(DEFAULT_COMPOSITION_CODE);
+    }
+  }, [isStdcm, speedLimitByTag, dispatchUpdateSpeedLimitByTag]);
```

Comment:
I think speedLimitByTag should not be a dependence of the useEffect, what if I select the tag 'no speed limit' ? Then the useEffect will set it to MA100.

I'm not sure it's the intended behaviour
```suggestion
  }, [isStdcm, dispatchUpdateSpeedLimitByTag]);
```

Reply:
Good catch ! I deleted it.